app/hallebarde/download_authorizer.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from hallebarde.infrastructure import exchange_repository
+from hallebarde.responses import authorizer_responses
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def handle(event: dict, context) -> dict:
+    logging.info('Processing event %s', str(event))
+    token = event['authorizationToken']
+    method_arn = event['methodArn']
+
+    policy = authorizer_responses.generate_policy(validate(token), method_arn)
+    return authorizer_responses.generate_response('hallebarde_download_authorizer', policy)
+
+
+def validate(upload_token: str) -> bool:
+    exchange = exchange_repository.get_by_download_token(upload_token)
+    return True if exchange else False
+
+
```

Comment:
```suggestion
```

Reply:
From flake8: `app/hallebarde/download_authorizer.py:23:1: W391 blank line at end of file`
You can find this feedback here, in the github actions CI pipeline: https://github.com/Dromadon/hallebarde/pull/28/checks?check_run_id=506458094#step:6:23